exporter/clickhouseexporter/README.md

```diff
@@ -342,3 +342,7 @@ service:
       processors: [ batch ]
       exporters: [ clickhouse ]
 ```
+
+## Developer tips
+
+- Make sure integration tests pass after any change of sql
```

Comment:
I don't think it's useful without instructions on running the integration tests. The integration tests should be run by the CI